techniques/general/control-orientation.html

```diff
@@ -1,30 +1,31 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 	<head>
-		<title>Using a control to allow access to content in different orientations which is otherwise restricted</title>
+		<title>Using a control to allow override of content orientation restrictions</title>
 		<link rel="stylesheet" type="text/css" href="https://rawgit.com/w3c/wcag21/master/css/sources.css"/>
 	</head>
 	<body>
-		<h1>Using a control to allow access to content in different orientations which is otherwise restricted</h1>
+		<h1>Using a control to allow override of content orientation restrictions</h1>
 		<section id="meta">
 			<h2>Metadata</h2>
 			<p id="id">ID: @@G210</p>
 			<p id="technology">Technology: General</p>
 			<p id="type">Type: Technique</p>
 		</section>
 		<section id="applicability">
-			<p>When the orientation of the page is locked, provide a button to allow a user to change the orientation.</p>
+			<p>When authors lock the orientation of the page, provide a control to allow a user to change the orientation.</p>
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>The objective of this technique is to allow users to access content in the way the user prefers. A content provider may expect that most users will view content using a specific device orientation or may expect that a user will want to maintain the original view within the device. As a result the provider then prevents the content from rotating. By providing a control to allow the user to rotate the content, someone who needs to use a particular orientation will be able to view the content in a comfortable manner. For example, a person who cannot hold the device and has it mounted attached to a wheelchair or bed.</p>
-      <p class="note">Many users will expect to use OS-level settings to prevent the orientation from changing. As a result, it is usually preferable to satisfy this Success Criterion by not restricting the content's ability to adapt to changes in orientation and allowing users to use the OS orientation lock when desired.</p>
+			<p>The objective of this technique is to allow users to override author orientation decisions and view content as they prefer. Authors may design content for a preferred orientation and expect that people will use this orientation when viewing their content, or the author may decide that a user wants to maintain the orientation with which they initiated viewing. As a result the author may prevent the content from rotating. However, this fails the Orientation Success Criterion, since the author has restricted the view and operation to one orientation.</p>
+			<p>By providing a prevalent mechanism to allow the user to rotate restricted content, the author meets the Orientation requirement. For example, a person who cannot hold the device and has it mounted to a wheelchair or bed will be able to re-orient the content to match their display's physical orientation.</p>
```

Comment:
Not sure about "a prevalent mechanism" here. We essentially want to say "By doing nothing to interfere with the operating system's ability to rotate content, the author meets the Orientation requirement."

Reply:
I agree entirely, but this whole technique appears to be about how the author, having not 'done nothing' can also still meet the SC. It's why I was concerned with the technique in the first place. In regard to the "prevalent mechanism" I figured we'd end up with some discussion about this. Fine taking it out since it hasn't previously been discussed, but to me this is very similar to an existing requirement (can't think what one it is right now) where an alternative version of something is available and it must be prevalent. In the case of orientation, if someone had to dig through 3 layers of submenu while trying to read the screen sideways in order to rotate the page, I think most folks would say that is really not accomplishing the SC.